shoop_workbench/settings/base_settings.py

```diff
@@ -30,7 +30,7 @@
     'filer',
     'easy_thumbnails',
     'shoop.core',
-    'shoop.simple_pricing',
+    'shoop.core.pricing',
```

Comment:
for some very strange reason this has re-appeared here, not sure why :) Will remove \o7